### PR TITLE
Release 0.8.0

### PR DESCRIPTION
**Pull request by lxgr-linux**

### CODE REVIEW

File: wiki.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-v0.7.2
+v088.0
```
Comment (MaFeLP) [inline wiki.md:1]: ?

**Review by MaFeLP — CHANGES_REQUESTED** [review]
Also, please update [README#ToDo](https://github.com/lxgr-linux/pokete/tree/release-1.8.0#todo), as it says there: `More than one Pokete for trainers` as an open ToDo

File: wiki.md (hunk 1 above)
Comment (lxgr-linux) [reply wiki.md]: First had a type in the version and then generated the wiki.

**Comment by lxgr-linux** [comment]
> Also, please update [README#ToDo](https://github.com/lxgr-linux/pokete/tree/release-1.8.0#todo), as it says there: `More than one Pokete for trainers` as an open ToDo

I will check this.

**Comment by lxgr-linux** [comment]
> Also, please update [README#ToDo](https://github.com/lxgr-linux/pokete/tree/release-1.8.0#todo), as it says there: `More than one Pokete for trainers` as an open ToDo

I will check this.

**Comment by lxgr-linux** [comment]
Done

**Review by MaFeLP — APPROVED** [review]
Looks good now, but please wait with merging, as flatpak needs the `assets/pokete.metainfo.xml` file from 0a410fba241504132fb803f99f3a9706fbe90ced, 95903fa2e735bbc299d8584a51c391dded8aab21 and 1a1089278a3601c803c99b083074c2e867dee00c. (I will take care or this)

BTW, do you want to use `assets/pokete.png` or `assets/pokete.svg` as the icon in the AppImage?

**Comment by lxgr-linux** [comment]
> BTW, do you want to use `assets/pokete.png` or `assets/pokete.svg` as the icon in the AppImage?

I think svg is always the better option, so yes.

**Pull request merged**